<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewProperty2</class>
 <widget class="QDialog" name="ViewProperty2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>352</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_nicknameView">
      <property name="text">
       <string>Property Nickname:</string>
      </property>
      <property name="buddy">
       <cstring>lineEdit_nickname</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="viewNickname">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelAddressView">
      <property name="text">
       <string>Address:</string>
      </property>
      <property name="buddy">
       <cstring>lineEdit_address</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="viewAddress">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_purchaseCostView">
      <property name="text">
       <string>Purchase Cost:</string>
      </property>
      <property name="buddy">
       <cstring>lineEdit_purchaseCost</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="viewPurchaseCost">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_purchaseDateView">
      <property name="text">
       <string>Purchase Date:</string>
      </property>
      <property name="buddy">
       <cstring>lineEdit_purchaseCost</cstring>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="labelMonthlyMortgageView">
      <property name="text">
       <string>Monthly Mortgage:</string>
      </property>
      <property name="buddy">
       <cstring>lineEdit_monthlyMortgage</cstring>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="viewMonthlyMortgage">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="labelMortgageTermView">
      <property name="text">
       <string>Mortgage Term in Years:</string>
      </property>
      <property name="buddy">
       <cstring>lineEdit_mortgageYears</cstring>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="viewMortgageTerm">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="labelHOAView">
      <property name="text">
       <string>HOA fee:</string>
      </property>
      <property name="buddy">
       <cstring>lineEdit_HOA</cstring>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="viewHOA">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="labelSewageCompanyView">
      <property name="text">
       <string>Sewage Company:</string>
      </property>
      <property name="buddy">
       <cstring>lineEdit_sewage</cstring>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="viewSewageCompany">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="labelSewageBilledTo">
      <property name="text">
       <string>Sewage Billed to:</string>
      </property>
      <property name="buddy">
       <cstring>lineEdit_nickname</cstring>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="viewSewageBill">
      <property name="text">
       <string/>
      </property>
      <property name="buddy">
       <cstring>lineEdit_nickname</cstring>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="labelUtilityView">
      <property name="text">
       <string>Utility Company</string>
      </property>
      <property name="buddy">
       <cstring>lineEdit_utility</cstring>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLabel" name="viewUtility">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="labelUtilitiesBilledTo">
      <property name="text">
       <string>Utilities Billed to:</string>
      </property>
      <property name="buddy">
       <cstring>lineEdit_nickname</cstring>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLabel" name="viewUtilityBill">
      <property name="text">
       <string/>
      </property>
      <property name="buddy">
       <cstring>lineEdit_nickname</cstring>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="labelTenantLNView">
      <property name="text">
       <string>Tenant Last Name</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QLabel" name="viewTenantLN">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="labelTenantEmailView">
      <property name="text">
       <string>Tenant Email:</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QLabel" name="viewTenantEmail">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="labelTenantPhoneView">
      <property name="text">
       <string>Tenant Phone Number:</string>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="QLabel" name="viewTenantPhone">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QLabel" name="labelMonthlyRentView">
      <property name="text">
       <string>Current Monthly Rent Charge:</string>
      </property>
     </widget>
    </item>
    <item row="15" column="1">
     <widget class="QLabel" name="viewTenantRent">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QLabel" name="labelBreakEvenView">
      <property name="text">
       <string>Break Even :</string>
      </property>
      <property name="buddy">
       <cstring>lineEdit_nickname</cstring>
      </property>
     </widget>
    </item>
    <item row="17" column="0">
     <widget class="QLabel" name="labelNotesView">
      <property name="text">
       <string>Notes:</string>
      </property>
      <property name="buddy">
       <cstring>lineEdit_notes</cstring>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="labelTenantFirstNameView">
      <property name="text">
       <string>Tenant First Name:</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLabel" name="viewTenantFN">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="viewPurchaseDate">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="16" column="1">
     <widget class="QLabel" name="viewBreakEven">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextBrowser" name="viewNotes">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
